client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -82,12 +98,12 @@ module.exports = React.createClass( {
 			return null;
 		}
 
-		tierViews = this.ratingTiers.map( function( tierLevel ) {
-			return this.renderRatingTier( tierLevel );
-		}, this );
+		const tierViews = this.ratingTiers.map( tierLevel => this.renderRatingTier( tierLevel ) );
 		return (
 			<div className="plugin-ratings">
-				<div className="plugin-ratings__rating-stars"><Rating rating={ this.props.rating } /></div>
+				<div className="plugin-ratings__rating-stars">
+					<Rating rating={ this.props.rating } />
+				</div>
 				<div className="plugin-ratings__rating-text">
 					{ this.translate( 'Based on %(ratingsNumber)s rating', 'Based on %(ratingsNumber)s ratings', {
 						count: this.props.numRatings,
```

Comment:
Totally optional, but you might find it useful to destructure the props used in each function; eg:

```js
render() {
  const { numRatings, rating } = this.props;
  // ...
}
```